content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -140,7 +140,7 @@ liveness-exec   1/1       Running   1          1m
 ## Define a liveness HTTP request
 
 Another kind of liveness probe uses an HTTP GET request. Here is the configuration
-file for a Pod that runs a container based on the `registry.k8s.io/liveness` image.
+file for a Pod that runs a container based on the `registry.k8s.io/e2e-test-images/agnhost:2.40` image.
```

Comment:
I'd drop the 2.40 here (makes updates easier IMO).

Reply:
Sadly there isn't a latest image available for this

2.40: https://explore.ggcr.dev/?image=registry.k8s.io%2Fe2e-test-images%2Fagnhost%3A2.40

No label, assume latest: https://explore.ggcr.dev/?image=registry.k8s.io%2Fe2e-test-images%2Fagnhost

Reply:
Here's what I mean:
```suggestion
file for a Pod that runs a container based on the `registry.k8s.io/e2e-test-images/agnhost` image.
```
If readers want to find a version to run manually, they'll have to either read on a few more lines (where they'll see it's 2.40), or query the registry.

Leave the actual manifest specifying the pinned version, and definitely don't use `latest` - we try hard _not_ to teach that!

Reply:
Perfect, thanks @sftim - will update this accordingly in the locale variations also.